### ISSUE TITLE
ways to avoid overfitting.

### ISSUE THREAD

**Issue by HuangBo-Terraloupe** [fizyr/keras-retinanet#434]
Hi, I got a question.

My project is a single category object detection problem, so the classification part is fine, what important is detection. 
I add validation generator into the training pipeline, and the model get overfitted after first epoch.
The validation regression loss is 2.2, it is still big.
And I did the evaluation on test dataset, the average IOU is 68%, I hope can increase the performance.
Is there any idea how to prevent over fitting? Should I add some dropout or other regularization method into network?
